contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -315,6 +334,29 @@ def _get_valid_version_for_repo(repo):
         return valid_versions
 
     @staticmethod
+    def _get_compatible_pack_version(repo, stackstorm_version):
+        candidates = {
+            'loose': None,
+            'strict': None
+        }
+        versions = DownloadGitRepoAction._get_valid_version_for_repo(repo)
```

Comment:
I actually changed this in master to remove leading "v" so it returns just a versions.

Anyway, minor refactoring should fix it - we can add `get_valid_version_tags_for_repo` which returns tags with "v" prefix and another "get_valid_versions_for_repo` which calls previous method and returns versions without "v" suffix.

I can look into this and push it directly into this branch since earlier today I just added some tests for this functionality :)